### ISSUE TITLE
Форме удаления аккаунта

### ISSUE THREAD

**Issue by Iryna-Ivasko** [msp4/shopper#68]
Это должна быть страница, где он должен отметить причину удаления, а также поля для воода текста  и удалить себя

**Comment by vladmokryi** [comment]
@Iryna-Ivasko пункт в профиле?

**Comment by Iryna-Ivasko** [comment]
@vladmokryi  ні, в меню її добаляти не потрібно, ми просто будемо висилати їм посилання, коли захочуть видалити профіль

**Comment by vladmokryi** [comment]
навіщо нам щось відсилати, якщо можна віддати цей функціонал їм

**Comment by Iryna-Ivasko** [comment]
Я думаю, що таким чином ми зменшимо к-ть видалень. Але кращу відповідь на це питання зможе дати alexzaporozhets

**Comment by vladmokryi** [comment]
@alexzaporozhets @orestx

**Comment by orestx** [comment]
В общем @Iryna-Ivasko правильно написала. Нужно что бы у клиента не было такой ссылки и мы могли давать ее по обращению в поддержку.

**Comment by vladmokryi** [comment]
>Нужно что бы у клиента не было такой ссылки
а по прямой ссылке доступ будет?

**Comment by Iryna-Ivasko** [comment]
@vladmokryi  
Причины удаления(radio button):
1. не было подходящих вакансий
2. нет желания быть ТП;
3. перехожу на другой, лучший сервис.
4. нестабильный доход
5. малая зарплата 
6. другое (напишу в комментарии)
+ поле textare (not required)

**Comment by Iryna-Ivasko** [comment]
1. не было подходящих вакансий
2. работа ТП не для меня;
3. нестабильный доход;
4. низкая оплата;
5. в доп. заработке уже не нуждаюсь;
6. теперь нет свободного времени;
7. ваш сервис не удобен в работе
8. другое (напишу в комментарии)
+ поле textare (not required)

**Comment by vladmokryi** [comment]
@Iryna-Ivasko 
- checkbox ? можно выбрать несколько ?
- textarea всегда активна или только при варианте №8 ?

**Comment by Iryna-Ivasko** [comment]
@vladmokryi да checkbox, але умова, щоб хоч одне з них було вибрано.  textarea активна завжди, але обов'язкова для заповнення тільки, коли активна умова 8.

**Comment by Iryna-Ivasko** [comment]
ось макет https://ninjastorage.blob.core.windows.net/html/FR2CT/index.html#44c3e088-a5db-9f91-c631-1eddf734b343

**Comment by Iryna-Ivasko** [comment]
Додаю скріни, бо при в експорті в html не все відобразилось
![image](https://cloud.githubusercontent.com/assets/12249275/9787769/0e583dcc-57ce-11e5-9c18-97e46f49e744.png)
![image](https://cloud.githubusercontent.com/assets/12249275/9787776/147a3c1e-57ce-11e5-94f6-9001f93f637f.png)

**Comment by vladmokryi** [comment]
![image](https://cloud.githubusercontent.com/assets/13148141/9797099/111872fa-5802-11e5-889e-eaa16754ad12.png)
@alexzaporozhets @Iryna-Ivasko @orestx

**Comment by alexzaporozhets** [comment]
нормально, эта страница для 3% пользователей
